logging/telemetry.go

```diff
@@ -134,9 +152,20 @@ func EnsureTelemetryConfig(dataDir *string, genesisID string) (TelemetryConfig,
 // EnsureTelemetryConfigCreated is the same as EnsureTelemetryConfig but it also returns a bool indicating
 // whether EnsureTelemetryConfig had to create the config.
 func EnsureTelemetryConfigCreated(dataDir *string, genesisID string) (TelemetryConfig, bool, error) {
+	/*
+		Our logic should be as follows:
+			- We first look inside the provided data-directory.  If a config file is there, load it
+			  and return it
+			- Otherwise, look in the global directory.  If a config file is there, load it and return it.
+			- Otherwise, if a data-directory was provided then save the config file there.
+			- Otherwise, save the config file in the global directory
+
```

Comment:
@tsachiherman do you know how we handle the deb/rpm installations? I thought there was another case where it could be stored somewhere in the /var directory

Reply:
I think that the `var` location was meant as a global location, and it's used by rpm installers.